src/Compilers/CSharp/Test/Emit/CSharpCompilerEmitTest.csproj

```diff
@@ -3,6 +3,7 @@
 <Project ToolsVersion="4.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <ImportGroup Label="Settings">
     <Import Project="..\..\..\..\..\build\Targets\VSL.Settings.targets" />
+	<Import Project="..\..\..\..\..\build\Targets\Roslyn.Toolsets.Xunit.targets" />
```

Comment:
We just went through a lot of work to combine our `Import` to a single.  Don't want to go back to having multiple ones.